src/components/helpers/touch_scroll_hook.ts

```diff
@@ -0,0 +1,94 @@
+import { Ref } from "../../types";
+import { useRefListener } from "./listener_hook";
+
+const friction = 0.95;
+
+const verticalScrollFactor = 1;
+
+const HorizontalScrollFactor = 1;
```

Comment:
These factors seem useless is the value is 1 ?

Reply:
I kept them in case i should fine tune through the testing phase